projects/opus/Dockerfile

```diff
@@ -21,3 +21,4 @@ RUN git clone https://gitlab.xiph.org/xiph/opus.git
 RUN wget https://opus-codec.org/static/testvectors/opus_testvectors.tar.gz
 WORKDIR opus
 COPY build.sh $SRC/
+COPY *.c *.cc *.options tests/
```

Comment:
if this really requires C++, we'll have to disable `dataflow` engine and sanitizer